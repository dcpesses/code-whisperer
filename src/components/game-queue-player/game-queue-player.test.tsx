import {render} from '@testing-library/react';
import GameQueue from './index';

describe('GameQueue', () => {
  test('Should render as expected', () => {
    const {container} = render(<GameQueue />);
    expect(container).toMatchSnapshot();
  });
});
