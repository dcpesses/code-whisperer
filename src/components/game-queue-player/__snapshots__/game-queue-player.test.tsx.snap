// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GameQueue > Should render as expected 1`] = `
<div>
  <div
    class="px-2 py-3 mb-0 small lh-1 border-bottom w-100 raleway-font fw-medium border rounded bg-dark-subtle"
  >
    <div
      class="d-flex justify-content-between"
    >
      <div>
        <strong
          class="text-body-emphasis fs-4 saira-condensed fw-bold"
        />
        <span
          class="text-info-emphasis d-block smaller fw-semibold"
        >
          55 years ago
        </span>
      </div>
      <button
        class="btn btn-secondary btn-sm fw-semibold"
      >
        Add to Playing
      </button>
    </div>
  </div>
</div>
`;
